content/docs/5.1/concepts/explore-tikv-features/replication-and-rebalancing.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Replication and rebalancing
+description: Replication and rebalancing
+menu:
+    "5.1":
+        parent: Features
+        weight: 1
+---
+
+This page walks you through a simple demonstration of how TiKV replicates, distributes, and rebalances data. Starting with a 3-node local cluster, you'll write some data via [go-ycsb](https://github.com/pingcap/go-ycsb) and verify that it replicates in triplicate by default. You'll then add 2 more nodes and watch how TiKV automatically rebalances replicas to efficiently use all available capacity. **NOTE: Do not apply this on production.**
+
+
+## Before you begin
+
+Make sure you have install [TiUP](https://github.com/pingcap/tiup) version **v1.5.2** or above.
+
+
+## Step 1. Start a 3-node cluster
+
+1. Use the `tiup-playground` to start a 3-node local cluster.
```

Comment:
```suggestion
Use the `tiup-playground` to start a 3-node local cluster.
```

Reply:
Though there is only one operation in this step, I am trying to keep the structure of all steps the same.

Reply:
Done.